scalingo.json

```diff
@@ -0,0 +1,7 @@
+{
+  "env": {
+    "IS_REVIEW_APP": {
+      "value": "true"
```

Comment:
ça le met uniquement sur les review app et ça ne le mettra pas sur l'instance de prod ni de qualif ?

Reply:
c'est ce que semble dire la doc
https://doc.scalingo.com/platform/app/review-apps#configuration-of-review-apps